crates/sargon/src/wrapped_radix_engine_toolkit/low_level/manifest_summary/account_deposit/lower_bound.rs

```diff
@@ -0,0 +1,90 @@
+use crate::prelude::*;
+
+/// Represents a lower bound on a non-negative decimal.
+///
+/// `NonZero` represents a lower bound of an infinitesimal amount above 0, and is included for
+/// clarity of intention. Considering Decimal has a limited precision of 10^(-18), it is roughly
+/// equivalent to an inclusive bound of 10^(-18), or Decimal::from_attos(1).
+#[derive(Clone, Debug, PartialEq, Eq, uniffi::Enum)]
+pub enum LowerBound {
+    NonZero,
+    Inclusive { decimal: Decimal },
```

Comment:
any doc from RET you can carry over to the varianets?

Reply:
updated docs